internal/services/cdn/cdn_frontdoor_custom_domain_resource_test.go

```diff
@@ -158,27 +142,17 @@ resource "azurerm_cdn_frontdoor_custom_domain" "import" {
 func (r CdnFrontDoorCustomDomainResource) update(data acceptance.TestData) string {
 	template := r.template(data)
 	return fmt.Sprintf(`
-provider "azurerm" {
-  features {}
-}
-
 %s
 
-resource "azurerm_dns_zone" "update" {
-  name                = "acctestzonealt%[2]d.com"
-  resource_group_name = azurerm_resource_group.test.name
-}
-
 resource "azurerm_cdn_frontdoor_custom_domain" "test" {
   name                     = "acctestcustomdomain-%[2]d"
   cdn_frontdoor_profile_id = azurerm_cdn_frontdoor_profile.test.id
-
-  dns_zone_id = azurerm_dns_zone.update.id
-  host_name   = join(".", ["%s", azurerm_dns_zone.test.name])
+  dns_zone_id              = azurerm_dns_zone.test.id
+  host_name                = join(".", ["sub-%[3]s", azurerm_dns_zone.test.name])
 
   tls {
     certificate_type    = "ManagedCertificate"
-    minimum_tls_version = "TLS12"
+    minimum_tls_version = "TLS10"
```

Comment:
why 1.0 and not 1.2?

Reply:
because that is the way it is declared in the APIs enum.

Reply:
because for the basic test it is set to 1.2, in the update test it is updating the value to 1.0 to validate functionality.

Reply:
makes sense to me